tests/src/rules/default.js

```diff
@@ -67,37 +67,37 @@ ruleTester.run('default', rule, {
 
     // from no-errors
     test({
-      code: "import Foo from './jsx/FooES7.js';",
-      parser: 'babel-eslint',
+      code: 'import Foo from \'./jsx/FooES7.js\';',
+      parser: require.resolve('babel-eslint'),
     }),
 
     // #545: more ES7 cases
     test({
-      code: "import bar from './default-export-from.js';",
-      parser: 'babel-eslint',
+      code: 'import bar from \'./default-export-from.js\';',
+      parser: require.resolve('babel-eslint'),
     }),
     test({
-      code: "import bar from './default-export-from-named.js';",
-      parser: 'babel-eslint',
+      code: 'import bar from \'./default-export-from-named.js\';',
+      parser: require.resolve('babel-eslint'),
     }),
     test({
-      code: "import bar from './default-export-from-ignored.js';",
+      code: 'import bar from \'./default-export-from-ignored.js\';',
       settings: { 'import/ignore': ['common'] },
-      parser: 'babel-eslint',
+      parser: require.resolve('babel-eslint'),
     }),
     test({
-      code: "export bar from './default-export-from-ignored.js';",
+      code: 'export bar from \'./default-export-from-ignored.js\';',
```

Comment:
Please revert these changes; the double quotes are intentional to avoid having to deal with escaping backslashes